core/src/main/java/org/wso2/testgrid/core/command/CleanUpCommand.java

```diff
@@ -66,11 +71,16 @@
             (ConfigurationContext.ConfigurationProperties.GRAFANA_DATASOURCE);
     private String grafanaApikey =
             ConfigurationContext.getProperty(ConfigurationContext.ConfigurationProperties.GRAFANA_APIKEY);
-
     private List<String> toDelete = new ArrayList<String>();
 
+    @Option(name = "--count",
+            usage = "Build Count",
+            aliases = { "-c" },
+            required = true)
+    private int remainingBuildCount = 100;
```

Comment:
I think we need to re-look at this API from the user's perspective. I do not think an end user would understand what is meant by 'count'. The usage of this API is as follows:

`testgrid cleanup --count=10`

This feels like we are going to 'clean' 10 items. But you actually meant to say, how many builds to keep.